### DISCUSSION TITLE
Is there any specific reason to prohibit "Settings" in custom module name?

### DISCUSSION THREAD

**Discussion by z0lo13** [YetiForceCompany/YetiForceCRM#17021] [General] [answered]
https://github.com/YetiForceCompany/YetiForceCRM/blob/56215d57aee22489a8852f53a627504377f76598/modules/Settings/ModuleManager/actions/Basic.php#L65

**Comment by rskrzypczak** [comment, marked as answer]
As for the name as a whole, yes it is.
But as part of a name, not necessarily.
This restriction was introduced to simplify verification of the "location" of the module, so you can't just skip this condition
